pytorch_lightning/trainer/properties.py

```diff
@@ -297,6 +297,11 @@ def distributed_sampler_kwargs(self):
 
         return kwargs
 
+    def print_colored_rank(self, msg):
```

Comment:
is this for debugging the draft or do you want to merge it?

if so, i'd do it separately

Reply:
Yes, I will remove it when when the PR is finished and submit a better version in another PR.